packages/cf-component-dropdown/test/Dropdown.js

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { trackFocus, getCurrentFocus } from '../../cf-test-focus/src/index';
```

Comment:
Maybe this can just be removed now? I didn't check if this was used in other places.